src/main/scala/sbtghactions/GenerativePlugin.scala

```diff
@@ -61,6 +61,7 @@ object GenerativePlugin extends AutoPlugin {
   private def isSafeString(str: String): Boolean =
     !(str.indexOf(':') >= 0 ||    // pretend colon is illegal everywhere for simplicity
       str.indexOf('#') >= 0 ||    // same for comment
+      str.indexOf('$') >= 0 ||    // and dollar sign
```

Comment:
Using wrap would imply the dollar sign should be escaped. this then cascades across all usages

Reply:
Why would dollar signs need to be escaped? They aren't special in YAML. I think I'm misunderstanding something. Do you have a value which contains `${{` and is *not* intended to be evaluated by GH Actions?

Reply:
Yes there is probably some confusion somewhere; but of I look at the checks run for this pull request those ${{ }}   ARE evaluated by GH Actions. So it seems it doesn't tell GH to avoid evaluating, but seems to be only important for yaml parsing... 

The mistake I wanted to address was assuming having it un-quoted in a list was valid YAML.. I only just started working with GH Actions myself, I'm not aware of how they are supposed to parse single quotes....

Reply:
Ah, so to clarify then, `$` isn't reserved in YAML at all, but it *is* interpreted specially by GitHub Actions itself, regardless of whether or not it's quoted. You need to double it in order to avoid being interpreted, but that's *usually* not what you want. My guess is you really do want `wrap` here (without the addition of the `$` reservation). If you have a runner which involves `$`, then you need to follow GH Actions' escaping explicitly within the specified value.

Reply:
It isn't reserved in YAML, but whatever GH is currently using the parse the file fails when given
```
[ ${{ matrix.os }} ]
```
or 
```
[ ${{ matrix.os }}, my-label ]
```
which was what I found out the hard way. 

And without that line, calling `wrap` will not output

`[ '${{ matrix.os }}', my-label ]` which makes GH happy

Reply:
> It isn't reserved in YAML, but whatever GH is currently using the parse the file fails when given

o_O That seems like a bug in GH Actions. I don't mind at all including a workaround here. Would you mind linking me to an example failure? In the meantime, I'm happy to merge this.

Reply:
I can't share the original repo, but here is a quick reproduction: 
[culprit change](https://github.com/kryptt/sbt-github-actions/commit/def8dee27116bbe3fd20f07e2e829bff7c7efd10)
[run log](https://github.com/kryptt/sbt-github-actions/actions/runs/253676397)

Reply:
Wow that is 100% a bug